### ISSUE TITLE
Configure name of totalCount and results in Json Result

### ISSUE THREAD

**Issue by abalad** [xtrinch/sails-pagination-middleware#1]
Would you like to set the return keys name?
{
 "totalCount": 0,
 "results": []
}

**Comment by xtrinch** [comment]
Sorry, what is the request? Dynamic naming of fields?

**Comment by abalad** [comment]
Yes. Is there any way to change the names? Because I have an API that has other names and swapping in FrontEnd would be a huge job heh

**Comment by xtrinch** [comment]
Not currently. It is a feature worth implementing though. I will look into it.

**Comment by abalad** [comment]
@xtrinch OK thank you.
